src/components/Dashboard.js

```diff
@@ -376,14 +389,14 @@ class Dashboard extends Component {
                 width={boxSizeWidth}
                 height={boxSizeHeight}
                 data={infoContainers}
-                legend={{ display: false }}
+                options={chartOptions}
               />
             </Paper>
           </div>
         </Row>
         <SnackComponent
           open={snackOpen}
-          duration={null}
+          duration={5000}
```

Comment:
Hey, @joserenatosilva! What do you think of setting this value (500) into a variable and use it here as a parameter?